scripts/core/helpers/url.ts

```diff
@@ -52,7 +52,22 @@ export const urlParams: ISuperdesk['browser']['location']['urlParams'] = {
     setString: (field, value) => {
         setUrlParameter(field, value?.length > 0 ? value : null);
     },
+    // Tags
+    getStringArray: (field) => {
```

Comment:
These two helper functions should be generic and deal with an Array of strings only, so they can be used elsewhere (outside SAMS). So the interface would be something like the following:
```typescript
interface urlParams {
    getStringArray(field: string): Array<string> | undefined;
    setStringArray(field: string, value: Array<string>): void;
}
```

Then in SAMS code you would convert before/after using these helper functions. i.e.

```typescript
// Get Tags from URL
const tags = (urlParams.getStringArray('tags') ?? []).map((tag) => ({
    code: tag,
    name: tag,
}));

// Set URL param for Tags
urlParams.setStringArray(
    'tags',
    (tags ?? []).map((tag) => tag.code)
);
```